docs/refguide/observer-component.md

```diff
@@ -1,6 +1,13 @@
 # @observer
 
-<a style="color: white; background:green;padding:5px;margin:5px;border-radius:2px" href="https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx">egghead.io lesson 1: observable & observer</a>
```

Comment:
Not sure if this link was intended to be the course page, I updated to link to the relevant lesson

Reply:
Nice catch!